modules/camera/camera_picamera2.py

```diff
@@ -11,11 +11,54 @@
     pass
 
 from . import base_camera
-from . import camera_configurations
 
 
-# TODO: pass in as constructor parameter
-CAMERA_TIMEOUT = 1
+class ConfigPiCamera2:
+    """
+    Configuration for the PiCamera.
+    """
+
+    def __init__(
+        self,
+        timeout: float = 1.0,
+        exposure_time: int = 250,
+        analogue_gain: float = 64.0,
+        contrast: float = 1.0,
+        maybe_lens_position: float | None = None,
+    ) -> None:
+        """
+        timeout: Getting image timeout in seconds.
+
+        exposure_time: Microseconds.
+        analogue_gain:
```

Comment:
A part of ISO, analogueGain * DigitialGain * 100 = ISO. From 0 to suggested max is 16, but can go up to 64.

Reply:
I'm not sure I understand the 2nd part. The acceptable range of values is 0 to 64, but the suggested range to use is 0 to 16 ? Why is the default argument 64 then?